src/login-init-app.js

```diff
@@ -25,10 +30,13 @@ export const store = createStore(
       sagaMiddleware,
       createLogger(),
       notifyAndDispatch,
+      routerMiddleware(history),
     )
   )
 );
 
+sagaMiddleware.run(rootSaga);
+
 /* Initial action */
```

Comment:
There is a lot going on in this init file and it's not too different from the other init files (dashboard-init-app, signup-init-app, init-plugin)...What do you think about the concept of breaking out these functions (e.g. store, initState, init_app) into their own files and using this init-app file just as a blueprint where we import the specific functions that are needed? 

Do you think it could work or do you see any immediate blockers?